### ISSUE TITLE
Option to remove default colorschemes

### ISSUE THREAD

**Issue by alichtman** [vim/vim#4399]
One feature I'd like to see in `vim` is the option to remove default colorschemes in a `.vimrc` file. For example, I will never use the `evening` colorscheme, but the only way to remove it from autocomplete options (that I am aware of) in the wildmenu is to [remove the colorscheme path](https://stackoverflow.com/a/11473883). The issue with this approach is that this is not persistent across updates. The accepted solution is writing a script to automate this. I don't think this is optimal.

I think there should be an option that could be added to a `.vimrc` like: `hidden_colorschemes=["evening",  ...]` that would remove these from the `colorschemes` cycling menu.

If there is a way to do this that I didn't find, please point me in the right direction. Thank you!

**Comment by chrisbra** [comment]
The wildmenu is there to show you all the possibilities of existing choices. I don't think hiding choices is a good thing to do (next user will ask why a certain choice does not appear). Rather use a custom completion, that can filter entries.

**Comment by alichtman** [comment]
> (next user will ask why a certain choice does not appear)

Maybe I didn't explain myself clearly. I intended this to be completely optional, and off by default. In a `.vimrc` file, you would be able to exclude certain colorschemes, however, you wouldn't be required to if you didn't want to. 

> The wildmenu is there to show you all the possibilities of existing choices.

My use case is removing some of the "existing choices" that the user makes the executive decision to never use.

> Rather use a custom completion, that can filter entries.

I googled around and didn't see a clear way to create a custom completion that would do what I was looking for, but I will continue looking for a solution. If you have any pointers / suggestions, they'd be much appreciated.
